src/getHumanAge.js

```diff
@@ -7,17 +7,26 @@
  * @returns {number[]}
  */
 function getHumanAge(catAge, dogAge) {
-  const catToHuman = catAge < 15
+  let catToHuman = catAge < 15
     ? 0
     : catAge < 24
       ? 1
       : 2 + Math.floor((catAge - 24) / 4);
-  const dogToHuman = dogAge < 15
+  let dogToHuman = dogAge < 15
     ? 0
     : dogAge < 24
       ? 1
       : 2 + Math.floor((dogAge - 24) / 5);
 
+  //
+  if (catAge >= 30) {
+    catToHuman = 0;
+  }
+
+  if (dogAge >= 30) {
+    dogToHuman = 0;
+  }
+
```

Comment:
why is this here? you shouldn't modify the solution, you should write only tests for it